Sources/StreamVideo/OpenApi/generated/Models/AggregatedStats.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+    
+public struct AggregatedStats: @unchecked Sendable, Codable, JSONEncodable, Hashable {
```

Comment:
Where is this one used?

Reply:
`GetCallStatsResponse`. `/video/call/{type}/{id}/stats/{session}`